src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -39,16 +53,21 @@ public ServiceManagerBuilder WithCallingAssembly()
         /// <returns>The instance of the <see cref="IServiceManager"/>.</returns>
         public IServiceManager Build()
         {
-            _options.ValidateOptions();
-
+            _services.AddSignalRServiceManagerCore();
```

Comment:
in general, method name with `Core` appended is a pretty private method that does not expect external calls